rewrite-yaml/src/test/java/org/openrewrite/yaml/YamlParserTest.java

```diff
@@ -129,4 +129,14 @@ void unicodeEscapes() {
           )
         );
     }
+
+    @Test
+    void projectBasedir() {
+    rewriteRun(
+        yaml(
+            """
+              server:
+                  keyStore: file:@project.basedir@/src/test/resources/keystore.jks
```

Comment:
Fails with
```
org.opentest4j.AssertionFailedError: [Bug in source parser or printer resulted in the following difference for "file.yaml"] 
expected: 
  "server:
      keyStore: 'file:@project.basedir@/src/test/resources/keystore.jks'"
 but was: 
  "server:
      keyStore: 'file:1fead784-d4ff-48b9-a9f6-34edbb297d12/src/test/resources/keystore.jks'"
```

Reply:
Tried a few iterations:
- no improvement when quoting the value
- no improvement when using `@foo@`
- `@@` works,
- `@foo` works, as does `foo@`